src/subarrayn.jl

```diff
@@ -0,0 +1,25 @@
+const unsafe_getindex = Base.unsafe_getindex
+
+function Base.isnull{T,N,P<:NullableArray,IV,LD}(V::SubArray{T,N,P,IV,LD}, I::Int...)
+    @Base._inline_meta
+    @boundscheck checkbounds(V, I...)
+    @inbounds r = V.parent.isnull[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
+    r
+end
+
+function Base.values{T,N,P<:NullableArray,IV,LD}(V::SubArray{T,N,P,IV,LD}, I::Int...)
+    @Base._inline_meta
+    @boundscheck checkbounds(V, I...)
+    @inbounds r = V.parent.values[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
+    r
+end
+
+@generated function anynull{T, N, U<:NullableArray}(S::SubArray{T, N, U})
```

Comment:
OK, for the moment, I put the new stuff into `subarraynew.jl`, and included it in `subarray.jl`.  That's the only way I could think of to completely avoid parsing (and executing the macros) the code for v0.5.  Is there a better technique that is recommended?  Thanks again for the review!